disperser/batcher/encoding_streamer_test.go

```diff
@@ -160,7 +160,7 @@ func TestBatchTrigger(t *testing.T) {
 	assert.Nil(t, err)
 	count, size := encodingStreamer.EncodedBlobstore.GetEncodedResultSize()
 	assert.Equal(t, count, 1)
-	assert.Equal(t, size, uint64(16384))
+	assert.Equal(t, size, uint64(26630))
```

Comment:
Occasional data point about the data size reduction with the new chunk encoding with Gnark(that's separately done): the  `16384` is the theoretical size (4 bytes/symbol * numSymbols in the chunk), and `26630` is the number of bytes from Gob encoding. The new chunk encoding will achieve the former.